phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/ConnectionInfo.java

```diff
@@ -525,6 +546,11 @@ protected String normalizeHostsList(String quorum, Integer defaultPort)
                     normalizedParts[i] = hostAndPort[0].trim().toLowerCase() + ":" + defaultPort;
                 } else if (hostAndPort.length == 2) {
                     normalizedParts[i] = quorumParts[i].trim().toLowerCase();
+                } else if (hostAndPort.length > 2 && hostAndPort[hostAndPort.length - 1].contains("]")) {
```

Comment:
This is a bit iffy.
Maybe we should just something else instead of '=' to replace the colons in the ipv6 address ?